src/layer/bridge.js

```diff
@@ -0,0 +1,123 @@
+"use strict";
+
+import * as Color from "../constants/color.js";
+import * as Util from "../js/util.js";
+
+// Bridge areas
+export const bridge = {
+  type: "fill-extrusion",
+  source: "openmaptiles",
+  "source-layer": "transportation",
+  id: "bridge",
+  minzoom: 13,
+  layout: {
+    visibility: "visible",
+  },
+  paint: {
+    "fill-extrusion-color": Color.bridgeFill,
+    "fill-extrusion-height": ["+", 3, ["coalesce", ["get", "layer"], 0]],
+    "fill-extrusion-opacity": 0.6,
+  },
+  filter: ["all", ["==", ["get", "class"], "bridge"]],
+};
+
+// Bridge casing for highways and railways
+export const bridgeCasing = {
+  type: "line",
+  source: "openmaptiles",
+  "source-layer": "transportation",
+  id: "bridge_casing",
+  minzoom: 13,
+  layout: {
+    "line-cap": "butt",
```

Comment:
This layer and the main road bridge layer need a line cap of `square`. Otherwise, the bridge area seems to extend beyond the roadway, obscuring a small segment of the roadway past either end of the bridge.

Reply:
I tried that and it looks weirder to me:

`"line-cap": "butt",`
![image](https://github.com/ZeLonewolf/openstreetmap-americana/assets/1732117/73066d48-0dbc-4e56-b68f-14ba7bf109f7)

`"line-cap": "square",`
![image](https://github.com/ZeLonewolf/openstreetmap-americana/assets/1732117/012ba4fa-94f6-49d4-8c50-ebc35d9fe3f3)

Reply:
Hm, maybe just the normal bridge roadway layer then?